spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -3009,6 +3009,179 @@ Mongo instance's configuration and logging routing.
 
 
 
+[[boot-features-neo4j]]
+=== Neo4j
+http://neo4j.com/[Neo4j] is an open-source NoSQL graph database that uses a
+rich data model of nodes related by first class relationships which is better 
+suited for connected big data than traditional rdbms approaches. 
+Spring Boot offers several conveniences for working with Neo4j, including the
+`spring-boot-starter-data-neo4j` '`Starter POM`'.
+
+[[boot-features-connecting-to-neo4j]]
+==== Connecting to a Neo4j database
+You can inject an auto-configured `org.neo4j.ogm.session.Neo4jSession` to
+access Neo4j databases.
+
+In your `application properties`, you can supply any domain packages to be scanned by the OGM at startup
+as well as the lifetime of the OGM session that will be established for web clients.
+
+By default your application will be configured to use an in-process embedded instance of Neo4j that will not persist any data when your application shuts down. You can also connect to a remote Neo4j server, or to an embedded instance that persists data between restarts of your application.
```

Comment:
Is that true? The starter does not bring the embedded driver so you need to add it yourself. What is the plan here? Should we add the embedded driver ourself? I think we should and that would be consistent with other areas where we support embedding. 

In this case, the documentation is wrong. Again, no need to act on the code but please explain the intend.

ping @jexp @vince-bickers